app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
```

Comment:
I'd rather see the xo configuration separately in the `package.json` as property. Same goes for the `babel` configuration. It's easier to add extra configuration and plugins.